tests/Integrations/MongoDB/Latest/composer.json

```
@@ -1,5 +1,5 @@
 {
     "require": {
-        "mongodb/mongodb": "1.21.1"
+        "mongodb/mongodb": "2.0.0"
```

```suggestion
        "mongodb/mongodb": "2.0.0"
```
TODO: Split Test suite